data/json/furniture_and_terrain/terrain-windows.json

```diff
@@ -737,24 +915,14 @@
   {
     "type": "terrain",
     "id": "t_metal_grate_window",
-    "name": "Window with metal grate",
-    "description": "Metal bars made into a grate for a window, Highly durability and will stand against any foe",
+    "name": "Barred reinforced boarded window",
```

Comment:
Please decapitalize all names.

Reply:
Done.

Reply:
> Please decapitalize all names.

Is this a style rule for terrain/furniture, or for items also?